tests/test_generics.py

```diff
@@ -104,10 +104,19 @@ def test_post_root_view(self):
         with self.assertNumQueries(1):
             response = self.view(request).render()
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertEqual(response.data, {'id': 4, 'text': 'foobar'})
+        self.assertEqual(response.data, {'id': 4, 'text': 'foobar', 'number': None})
         created = self.objects.get(id=4)
         self.assertEqual(created.text, 'foobar')
 
+    def test_post_null_integer_as_form_data(self):
+        """
+        None should resolve when a model's field has null=True.
+        """
+        data = {'text': 'foobar', 'number': None}
+        request = factory.post('/', data)
+        response = self.view(request).render()
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
```

Comment:
Yeah, I wanted to put it in test_testing.py, but the fixtures/views there didn't really suit my needs.  I see you've done a lot of refactoring with these tests since I last visited.  Still trying to understand the new style and find the best place to integrate.